assignment-01.md

```diff
@@ -1,30 +1,101 @@
+https://hrmars.com/papers_submitted/3512/Islamic_Law_on_Gambling_and_Some_Modern_Business_Practices.pdf
 
-3. List down 3 Non-Shariah Compliant Business Activities. 
+**3. List down 3 Non-Shariah Compliant Business Activities.** <br/>
+       1) Pork(along with all byproducts of pigs)<br/>
+       2) Conventional insurance(the Shariah Compliant alternative of the insurance is known as Takaful)<br/>
+       3) Gambling<br/>
 
-4. Name the stock exchange of Malaysia.
+References:<br/>
+https://av.sc.com/my/content/docs/Non-Syariah-Compliant-Business-Activities_ref.pdf
 
-5. Create a new folder name 'AR' in your _repo main branch_. Upload Annual Report 2018 - 2020 for companies below: <br>
+**4. Name the stock exchange of Malaysia.**<br/>
+Bursa Malaysia(KLSE). 
+
+References: <br/>
+Nil
+
+**5. Create a new folder name 'AR' in your _repo main branch_. Upload Annual Report 2018 - 2020 for companies below:** <br>
 (a) ViTrox Corp <br>
 (b) Pentamaster Corp <br> 
 *There should be at least 6 PDF files. <br>
 
-6. Who are the users of annual reports?
+References: <br/>
+https://www.vitrox.com/investor/annual-report.php
+https://pentamaster.listedcompany.com/ar.html
+
+**6. Who are the users of annual reports?** <br/>
+
+The shareholder and the stakeholders of a particular company. The annual report allows the stockholders to determine the company's growth and current state, along with its future projection. This provides information to the the shareholders' decision making process, so that they can evaluate their investment portfolio and make changes if deemed suitable.
+
+References: <br/>
+https://www.investopedia.com/terms/a/annualreport.asp#:~:text=The%20intent%20of%20the%20required,and%20to%20make%20investment%20decisions.
 
-7. What is the most important part of an annual report?
+**7. What is the most important part of an annual report?** 
 
-8. You have RM 10,000 savings and want to purchase shares as an investment for earning side income. 
+The financial statement is usually considered the most important, mostly because it shows the planning and foresight of the company for future growth, the ability of a company to handle debts, along with the past performance of the company. The financial statement features balance sheet and account statement, which provides information on the revenue, profit and losses of the company.
+
+References: <br/>
+https://www.freshbooks.com/hub/reports/write-an-annual-report#:~:text=The%20financial%20statements%20are%20the,and%20its%20plans%20for%20growth.
+https://www.investopedia.com/terms/i/incomestatement.asp
+
+**8. You have RM 10,000 savings and want to purchase shares as an investment for earning side income. 
 You have listed two potential companies and need to select one company to invest. 
-Comparing _Pentamaster Corporation Bhd._ and _ViTrox Corporation Bhd._ Justify your investment choice.
+Comparing _Pentamaster Corporation Bhd._ and _ViTrox Corporation Bhd._ Justify your investment choice.** <br/>
```

Comment:
Impressive attempt! Seems like you grasped the idea of financial analysis very well! Surely AKW104 won't be difficult for you! Well done.

Reply:
Thank you for the feedback!